ibm/service/kubernetes/resource_ibm_container_vpc_cluster.go

```diff
@@ -945,13 +945,14 @@ func resourceIBMContainerVpcClusterRead(d *schema.ResourceData, meta interface{}
 		return fmt.Errorf("[ERROR] Error retrieving conatiner vpc cluster: %s", err)
 	}
 
-	workerPool, err := csClient.WorkerPools().GetWorkerPool(clusterID, "default", targetEnv)
+	wps, err := csClient.WorkerPools().ListWorkerPools(clusterID, targetEnv)
+	if err != nil || len(wps) == 0 {
+		return fmt.Errorf("[ERROR] Error retrieving workerpools of the cluster %s: %s", clusterID, err)
+	}
+	wp := wps[0]
```

Comment:
Probably would be better to remove this zones read. Previously it only read the "default" which had the same issue, on top of that it could be removed. (which is an issue that this PR is trying to solve)
The fields it fills are set with ApplyOnce so shouldn't carry weight anymore.

There is dedicated workerpool resource to be used and read from.

Reply:
Fair enough